packages/@lwc/engine/src/framework/template.ts

```diff
@@ -211,7 +212,7 @@ export function evaluateTemplate(vm: VM, html: Template): Array<VNode | null> {
                 // Set the global flag that template is being updated
                 isUpdatingTemplate = true;
 
-                vnodes = html.call(undefined, api, component, cmpSlots, context.tplCache!);
+                vnodes = html.call(undefined, api, component, context.tplCache!);
```

Comment:
we not longer send the allocated `cmpSlots` in engine, so, we don't pass it to the template function anymore.